### PR TITLE
js-feat-summary-table

### PR DESCRIPTION
**Pull request by sampoonachot**

### CODE REVIEW

File: client/assets/icon-moneytime.svg [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	.st1{fill:none;stroke:#A6C88E;stroke-width:1.5;stroke-linecap:round;}
+</style>
+<title>icon_moneytime</title>
+<desc>Created with Sketch.</desc>
+<g id="Build-Form-Details">
+	<g id="B01" transform="translate(-898.000000, -611.000000)">
+		<g id="Modal" transform="translate(33.000000, 32.000000)">
+			<g id="Fixed-Sidebar---m1.small" transform="translate(773.000000, 139.000000)">
+				<g id="Configuation-Details">
+					<g id="icon_moneytime" transform="translate(93.000000, 441.000000)">
```
Comment (rawfalafel) [inline client/assets/icon-moneytime.svg]: what time is it?

Comment (rawfalafel) [reply client/assets/icon-moneytime.svg]: MONEY TIME

File: client/styling/summary-table.less [hunk 2]
```diff
@@ -0,0 +1,160 @@
+@changedFontColor: @blue;
+@defaultSpacing: 15px;
+@smallFontSize: 13px;
+@normalFontSize: 16px;
+@largeFontSize: 24px;
```
Comment (rawfalafel) [inline client/styling/summary-table.less]: This variable name is actually a global variable name. Its going to get clobbered with `@largeFontSize: 26px` in your other PR. I'd either rename this to `summaryTableLargeFontSize` or unify the font sizes or get rid of the variable altogether.

File: client/assets/icon-moneytime.svg (hunk 1 above)
Comment (sampoonachot) [reply client/assets/icon-moneytime.svg]: el oh el

File: client/styling/summary-table.less (hunk 2 above)
Comment (sampoonachot) [reply client/styling/summary-table.less]: ok, i unified them

**Pull request merged**